src/pages/home/ReportScreen.js

```diff
@@ -312,9 +318,15 @@ class ReportScreen extends React.Component {
                             {/* Note: The report should be allowed to mount even if the initial report actions are not loaded. If we prevent rendering the report while they are loading then
                             we'll unnecessarily unmount the ReportActionsView which will clear the new marker lines initial state. */}
                             {(!this.isReportReadyForDisplay() || isLoadingInitialReportActions) && (
-                                <ReportActionsSkeletonView
-                                    containerHeight={this.state.skeletonViewContainerHeight}
-                                />
+                                <>
+                                    <ReportActionsSkeletonView
+                                        containerHeight={this.state.skeletonViewContainerHeight}
+                                    />
+                                    <ReportFooter
```

Comment:
While the report is loading, user interaction in composer is disabled. (Implemented by this [commit](https://github.com/Expensify/App/commit/6fe2785f674a5a6933663a88dc47c02a6d336ea9)) We have [a feature request](https://github.com/Expensify/App/issues/14223) to enable composer even when the report is loading. So, @0xmiroslav @rushatgabhane  I am trying to understand why was it disabled? 

cc: @Beamanator @marcaaron

Reply:
Yes, we chose to keep it disabled as it was not displayed at all before. So basically, we decided to focus on just one ticket at a time rather than open a big box of problems 😄